gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/converter/CategoryMapper.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.service.converter;
+
+import static java.util.Objects.nonNull;
+
+import io.gravitee.rest.api.model.CategoryEntity;
+import io.gravitee.rest.api.service.CategoryService;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+
+/**
+ * @author Sergii ILLICHEVSKYI (sergii.illichevskyi at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Slf4j
+@Component("Console")
```

Comment:
I can't. If I delete it produce this error
`Caused by: org.springframework.context.annotation.ConflictingBeanDefinitionException: Annotation-specified bean name 'categoryMapper' for bean class [io.gravitee.rest.api.portal.rest.mapper.CategoryMapper] conflicts with existing, non-compatible bean definition of same name and class [io.gravitee.rest.api.service.converter.CategoryMapper]`

Meaning that there are 2 CategoryMappers. One for Portal and one for Console.

I can rename this class, but it will introduce changes in a lot of classes that use it. LIke 30 classes.
Which I don't want to do since I will be patching it to 4 legacy branches.

But this way - if I add an annotation, Spring will distinguish it from the other CategoryMapper in Portal.
And I would not need to modify classes that use it.